src/components/TodoForm/TodoForm.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TodoForm = ({ onChangeHandler, onSelectHandler, inputValue}) => (
+  <div className="ui form">
+    <div className="two fields">
+      <div className="field">
+        <label>
+          Title
+          <input
+            placeholder="Enter task title"
+            type="text"
+            value={inputValue}
+            onChange={event => onChangeHandler(event)}
```

Comment:
запомни, пожалуйста, этот момент, исправь везде
```suggestion
            onChange={onChangeHandler}
```

Reply:
виправив